### PR TITLE
AC Convert Test - LPD-32222 and LPD-32223 | master

### PR DESCRIPTION
**Pull request by ViniLopes20**
https://liferay.atlassian.net/browse/LPD-32222
https://liferay.atlassian.net/browse/LPD-32223
[Testray](https://testray.liferay.com/#/project/34880/routines/730294/build/44701249)

### CODE REVIEW

**Comment by ViniLopes20** [comment]
ci:test:analytics-cloud

**Comment by ViniLopes20** [comment]
ci:test:sf

File: modules/test/playwright/tests/osb-faro-web/individuals.spec.ts [hunk 1]
```diff
@@ -56,6 +51,7 @@ test(
 			});
 		});
 
+		const date = new Date();
```
Comment (marcosapmf) [inline modules/test/playwright/tests/osb-faro-web/individuals.spec.ts:54]: Where are we using this variable?

**Review by marcosapmf — APPROVED** [review]

**Comment by ViniLopes20** [comment]
ci:forward

**Comment by ViniLopes20** [comment]
ci:test:sf

**Comment by ViniLopes20** [comment]
ci:forward

**Comment by ViniLopes20** [comment]
ci:forward

**Comment by ViniLopes20** [comment]
Covered by: https://github.com/liferay-ac/liferay-portal/pull/1706

**Pull request closed without merging**